.github/workflows/Benchmarks.yml

```diff
@@ -17,17 +17,39 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        build: [windows-2022-release, linux-kvm-release, linux-hyperv-release]
+        build: [
+          windows-2022-release-amd,
+          linux-kvm-release-amd,
+          linux-hyperv-release-amd,
+          windows-2022-release-intel,
+          linux-kvm-release-intel,
+          linux-hyperv-release-intel,
+        ]
         include:
-          - build: windows-2022-release
+          - build: windows-2022-release-amd
             os: [self-hosted, Windows, X64, "1ES.Pool=hld-win2022-amd"]
             hypervisor: none
```

Comment:
Optional: since we are refactoring some ci here, could we make the hypervisor different from `none` on windows? I'm thinking kvm/mshv/hyperv

Reply:
Changed the Windows name to "hyperv". Don't see a reason to change the Linux name, feel like that could lead to confusion.